app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -380,6 +384,32 @@ def handle_successful_status_update(enrollment, response)
     )
   end
 
+  def handle_call_in_needed(enrollment, response)
+    proofed_at = parse_usps_timestamp(response['transactionEndDateTime'])
+    enrollment_outcomes[:enrollments_passed] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
+      **enrollment_analytics_attributes(enrollment, complete: true),
+      **response_analytics_attributes(response),
+      passed: true,
+      reason: 'Successful status update',
+      job_name: self.class.name,
+    )
+    enrollment.profile.activate_after_passing_in_person
```

Comment:
I think we actually want to fold this into my `handle_fraud_review_pending` method (in my LG-11997 PR). I don't think we want to activate them.

Reply:
i'll delete this line